### PR TITLE
Feature/fingerbank manufacturer info

### LINKED ISSUE #3211: Fingerbank info: Keep the hardware manufacturers when there is DHCP data update on a node
Title says it all, king Semaan knows it as well.

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Display the hardware manufacturer info in the Fingerbank tab

# Impacts
Fingerbank integration

# NEW Package(s) required
fingerbank-4.0.1
fingerbank-collector-1.0.1

# Issue
fixes #3211 

# Delete branch after merge
YES

# NEWS file entries
## Enhancements
* Support displaying the device manufacturer in the Fingerbank node tab

### CODE REVIEW

**Comment by extrafu** [comment]
Looks good to me. @jrouzierinverse please check the DAL part and merge if all fine.

**Comment by cgx** [comment]
I know we haven't been consistent in the past, but we should capitalize the first letter of each word in labels. So let's use *Device Manufacturer* everywhere.

**Comment by julsemaan** [comment]
Both new packages for RHEL are in the devel repo

**Comment by julsemaan** [comment]
@cgx 

Casing fixed

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 1]
```diff
@@ -50,4 +50,7 @@ call ValidateVersion;
 ALTER TABLE node
     MODIFY device_score int DEFAULT NULL;
 
+ALTER TABLE node
+    ADD COLUMN device_manufacturer VARCHAR(255) AFTER device_score;
+
```
Comment (jrouzierinverse) [inline db/upgrade-X.X.X-X.Y.Z.sql]: Merge this with the previous ALTER TABLE of node.

**Review by jrouzierinverse — CHANGES_REQUESTED** [review]

**Comment by julsemaan** [comment]
Adjusted based on comments, ready for another review/merge

**Pull request merged**